pages/en/posts/google-unity-partner-to-support-chromebooks.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Google & Unity partner to support Chromebooks
+metadesc: Google is partnering with Unity to bring x86 build support and input and windowing improvements starting in Unity 2021.2.0a14.
+tags:
+  - product news
+  - gaming
+  - unity
+  - keyboard support
+  - mouse support
+  - large screen
```

Comment:
```suggestion
  - large screens
```
nit, but I think the tag is actually "large screens" plural

Reply:
Oh good catch!